src/components/measurements/measurementsD3.js

```diff
@@ -197,7 +197,7 @@ export const drawMeasurementsSVG = (ref, xAxisRef, svgData) => {
   drawStickyXAxis(xAxisRef, containerHeight, svgHeight, xScale, x_axis_label);
 
   // Add threshold(s) if provided
-  if (thresholds !== null) {
+  if (thresholds !== null && thresholds !== undefined) {
     for (const threshold of thresholds) {
```

Comment:
_Overthinking nit; non-blocking_

Since you're making this change, I wonder if it's worth adding an explicit check for `thresholds` being iterable? If, somehow, `thresholds === 3`, L201 is going to throw `Uncaught TypeError: threasholds is not iterable`.

Reply:
Fair! Better to just check that `thresholds` is an array.

Reply:
Updated in https://github.com/nextstrain/auspice/pull/1802/commits/172c55622a12e2838c0f106f2280f17be399791a